integration/local/utils.mjs

```diff
@@ -0,0 +1,41 @@
+// ChainSafe/dappeteer's metamask methods mostly doesn't not work out of box, fml
```

Comment:
👍

Reply:
Hello! These are really helpful. Have you thought about contributing these back to the chainsafe/dappeteer project?